src/main/java/orionmipt/Context.java

```diff
@@ -0,0 +1,309 @@
+package orionmipt;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+
+class Context implements Cloneable {
+
+    private final Map<Object, String> classAttributeMap;
```

Comment:
You can initialize these variables right here (`classAttributeMap = new HashMap<>()`)